src/pybind/mgr/dashboard/services/progress.py

```diff
@@ -9,12 +9,11 @@
 
 from __future__ import absolute_import
 
-from datetime import datetime
 import logging
+from datetime import datetime
 
-from . import rbd
 from .. import mgr
-
+from . import rbd  # pylint: disable=no-name-in-module
```

Comment:
Don't know the reason why pylint throws this error here.

Reply:
This seems to fix the problem:
```
diff --git a/src/pybind/mgr/dashboard/.pylintrc b/src/pybind/mgr/dashboard/.pylintrc
index 222bfe18d01..f7593179a09 100644
--- a/src/pybind/mgr/dashboard/.pylintrc
+++ b/src/pybind/mgr/dashboard/.pylintrc
@@ -16,7 +16,8 @@ ignore-patterns=
 
 # Python code to execute, usually for sys.path manipulation such as
 # pygtk.require().
-#init-hook=
+init-hook='import sys; sys.path.append("./")'
+
 
 # Use multiple processes to speed up Pylint.
 jobs=1
```

found it here: https://stackoverflow.com/a/3065082

Reply:
It seems I replied on the wrong comment.
This change fixes the need for `# pylint: disable=import-error`.